source/d3d8to9_base.cpp

```diff
@@ -10,10 +10,11 @@ static const D3DFORMAT AdapterFormats[] = {
 	D3DFMT_X8R8G8B8,
 	D3DFMT_R5G6B5,
 	D3DFMT_X1R5G5B5,
-	D3DFMT_A1R5G5B5,
-	D3DFMT_A2R10G10B10
+	D3DFMT_A1R5G5B5
 };
 
+IDirect3DDevice9 *pCurrentDeviceInterface = nullptr;
```

Comment:
This should be fairly easy to check via a simple unit test, I guess? As it's COM, you probably can't have multiple pointers to the same class and thus if two Direct3DCreate8 calls return the same pointer, then it means it's the same device.